src/addresses/Fraxtal_L2_Devnet.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: ISC
+pragma solidity >=0.8.0;
+
+// **NOTE** Generated code, do not modify.  Run 'npm run generate:constants'.
+
+import { TestBase } from "forge-std/Test.sol";
+
+library Canonicals {
+    address internal constant FRAX = 0xFc00000000000000000000000000000000000001;
+    address internal constant FXS = 0xFc00000000000000000000000000000000000002;
+    address internal constant FPI = 0xFc00000000000000000000000000000000000003;
+    address internal constant FPIS = 0xfc00000000000000000000000000000000000004;
+    address internal constant FRXBTC = 0xfC00000000000000000000000000000000000007;
+    address internal constant WFRXETH = 0xFC00000000000000000000000000000000000006;
+    address internal constant SFRXETH = 0xFC00000000000000000000000000000000000005;
+    address internal constant SFRAX = 0xfc00000000000000000000000000000000000008;
+}
+
+library Farms {}
+
+library FraxBonds {}
+
+library FraxFerry {}
+
+library FraxlendAndOracles {}
+
+library Fraxswap {}
+
+library Misc {
+    address internal constant FRAXCHAIN_ADMIN = 0x8b1E899ec5d51387a41Ee68261149F7d3Dfd7160;
+    address internal constant L2_STANDARD_BRIDGE = 0x4200000000000000000000000000000000000010;
+    address internal constant PROXY_ADMIN = 0xfC00000000000000000000000000000000000009;
+}
+
+library Multisigs {
+    address internal constant PROXY_ADMIN = 0xfC00000000000000000000000000000000000009;
+    address internal constant COMPTROLLER = 0x8b1E899ec5d51387a41Ee68261149F7d3Dfd7160;
```

Comment:
This is not a msig, it's an EOA, I think it's ok to leave it there tho, lmk what you think

Reply:
Will leave it alone